curve25519-dalek/src/backend/serial/u64/scalar_specs.rs

```diff
@@ -50,12 +50,50 @@ pub open spec fn five_limbs_to_nat_aux(limbs: [u64; 5]) -> nat {
     pow2(208) * (limbs[4] as nat)
 }
 
+/// Non-recursive version of bytes_to_nat for 32 bytes
+/// This is the non-recursive equivalent, similar to as_nat_32_u8 in field_core.rs
+pub open spec fn bytes_to_nat_aux(bytes: &[u8; 32]) -> nat {
```

Comment:
This is the same as https://github.com/Beneficial-AI-Foundation/dalek-lite/blob/9c8605e00ffc273bdfb76e0fba088b6681388c83/curve25519-dalek/src/backend/serial/u64/field_lemmas/field_core.rs#L43
We should move that one from `field_core` to a shared `backend_64_core` (or some such) and reuse instead of duping.